spec/services/discovery/geoblacklight_document_spec.rb

```diff
@@ -87,6 +87,23 @@
         expect(document[:dc_rights_s]).to eq 'Public'
       end
     end
+
+    context 'with a public image work with a mapset parent' do
+      let(:identifier) { ['ark:/99999/fk44jq866'] }
+      let(:map_set) { FactoryGirl.build(:map_set_with_image_work, id: 'map-set-1', identifier: identifier) }
+      let(:visibility) { Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC }
+
+      before do
+        map_set.ordered_members << geo_concern
+        map_set.save
+        geo_concern.update_index
+      end
+
+      it 'returns a suppressed document with a source field' do
+        expect(document[:suppressed_b]).to eq true
```

Comment:
can you add a test that asserts that a mapset would exclude the suppressed field?